metadetect/mfrac.py

```diff
@@ -0,0 +1,49 @@
+import ngmix
+import numpy as np
+
+from .shearpos import DEFAULT_STEP
+from .detect import CatalogMEDSifier
+from .defaults import BMASK_EDGE
+
+
+def measure_mfrac(
+    *,
+    mfrac,
+    x,
+    y,
+    box_sizes,
+    obs,
+    fwhm,
+    step=DEFAULT_STEP,
+):
+    if fwhm is None:
+        fwhm = 1.2
+
+    obs = obs.copy()
+    obs.set_image(mfrac)
+
+    gauss_wgt = ngmix.GMixModel(
+        [0, 0, 0, 0, ngmix.moments.fwhm_to_T(fwhm), 1],
+        'gauss',
+    )
+    mbobs = ngmix.MultiBandObsList()
+    obslist = ngmix.ObsList()
+    mbobs.append(obslist)
+    obslist.append(obs)
+    m = CatalogMEDSifier(mbobs, x, y, box_sizes).get_meds(0)
```

Comment:
why is this needed instead of our "usual" medsifier?

Reply:
what I mean is we already have a medsifier at this point right?  Why not re-use it?